### DISCUSSION TITLE
cursor stops on it's own

### DISCUSSION THREAD

**Discussion by offlxnz** [ppy/osu#19944] [Q&A] [answered]
### Type

Other

### Bug description

just installed lazer, wanted to play it with tablet, but sometimes cursor stops on it's own not sure if it's a bug or issue with my tablet (huion h420)

### Screenshots or videos

https://user-images.githubusercontent.com/107271606/186510332-2c98850a-5a27-4c2a-aa9d-1659cb420562.mp4

### Version

2022.821.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/9419056/network.log)
[performance.log](https://github.com/ppy/osu/files/9419057/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9419058/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9419059/updater.log)
[database.log](https://github.com/ppy/osu/files/9419060/database.log)

**Comment by bdach** [comment, marked as answer]
This looks like not only your mouse is hanging, but the entire game is stop-starting.

* Do you have any input devices (mice, especially) that poll at or above 1000Hz?
* Please attach a video of this with the frame graph showing (ctrl-f11 twice).

I've converted this to a discussion because this is 99% something specific to your setup.

**Reply by offlxnz** [reply]
at the moment i have only my tablet and mice connected at the same time, could it be because of this?
and the video that you asked for(i'm moving my cursor with tablet, but when i move it with mouse, all becomes normal)


https://user-images.githubusercontent.com/107271606/186511865-d75b6821-ae57-44a6-8b40-c0617d11de1d.mp4

**Reply by bdach** [reply]
A few more diagnostic questions:

* Does this happen when _only_ the tablet is connected, without the mouse?
* Do you have any external tablet drivers installed that run outside of the game?
* Does toggling the tablet input in game settings help?

**Reply by offlxnz** [reply]
1)yes
2)i have only manufacturer driver, because my tablet don't support opentabletdriver 
3)it doesn't help, it doesn't even see tablet

**Reply by bdach** [reply]
Ok, can you try one more thing - do you have "high precision mouse" enabled? Does disabling it fix it?

**Reply by offlxnz** [reply]
yes, it helped, thank you very much

**Reply by peppy** [reply]
Tracking at https://github.com/ppy/osu/issues/19960.
